Refresh.Interfaces.Game/Endpoints/ReviewEndpoints.cs

```diff
@@ -124,13 +125,21 @@ public Response PostReviewForLevel(RequestContext context,
         if (!database.HasUserPlayedLevel(level, user))
             return BadRequest;
 
+        IEnumerable<Label> labels = [];
+
+        if (!string.IsNullOrWhiteSpace(body.Labels))
+        {
+            // Make sure there aren't too many and duplicate labels.
+            labels = LabelExtensions.FromLbpCommaList(body.Labels).Distinct().Take(UgcLimits.MaximumLabels);
```

Comment:
Try to avoid chaining LINQ on a single line like this, put each call on a separate line.